### ISSUE TITLE
SD.h wrong calculation of totalBlocks

### ISSUE THREAD

**Issue by erpebe** [earlephilhower/arduino-pico#1893]
In the following code should that not be a multiplication instead of a division?
There are more blocks than clusters.

```
size_t totalBlocks() {
  return (totalClusters() / blocksPerCluster());
}
```

**Comment by earlephilhower** [comment]
Seems like a good catch.  Want to do the PR since you found and fixed it?

**Comment by earlephilhower** [comment]
Fixed by #1899
